src/styles/sidebar/components/annotation.scss

```diff
@@ -61,6 +61,13 @@
   padding-top: 0.5em;
 }
 
+.annotation-link-more {
+  display: flex;
+  flex-flow: row-reverse;
+  padding-bottom: 5px;
+  padding-top: 5px;
+}
```

Comment:
This partially duplicates another CSS selector which appears on line 49.

Reply:
I think this was merge conflict typo I made